app/scripts/modules/core/cluster/task.matcher.ts

```diff
@@ -0,0 +1,92 @@
+import {has, intersection, map} from 'lodash';
```

Comment:
I pulled these out of the original cluster service, but made it a plain old file instead of an Angular module, since there's nothing injected and I don't imagine we'll ever inject anything Angular into it, in light of recent events.